partition/roles/sonic/templates/metal.yaml.j2

```diff
@@ -107,15 +107,23 @@ PORT:
     {% if 'parent_port' in running_cfg %}
     parent_port: {{ running_cfg.parent_port }}
     {% endif %}
+    admin_status: up
     {% if sonic_ports_dict[name] is defined %}
     {% set port = sonic_ports_dict[name] %}
-    admin_status: up
```

Comment:
Why was this changed?

Reply:
To make deployments less verbose. Otherwise you have to explicitly add every port to the `sonic_ports_dict`, where you want to have `admin_status: up`.

Reply:
Could you allow to deactivate a port?